### PR TITLE
PREAPPS-6698:Modern UI - Users should not be allowed to use username in the password

### PR DESCRIPTION
**Pull request by sakshirawat0247**
-Added zimbraPasswordAllowUsername to the schema of AccountInfoAttrs

### CODE REVIEW

File: src/schema/generated-schema-types.ts [hunk 1]
```diff
@@ -129,6 +129,7 @@ export type AccountInfoAttrs = {
   zimbraMtaMaxMessageSize?: Maybe<Scalars['Float']>;
   zimbraPasswordAllowedChars?: Maybe<Scalars['String']>;
   zimbraPasswordAllowedPunctuationChars?: Maybe<Scalars['String']>;
+	zimbraPasswordAllowUsername?: Maybe<Scalars['Boolean']>;
```
Comment (silentsakky) [inline src/schema/generated-schema-types.ts:132]: is this manually added? this shouldn't have indentation issue as this is auto generated file

Comment (sakshirawat0247) [reply src/schema/generated-schema-types.ts]: updated the file again with correct auto generation

**Review by silentsakky — APPROVED** [review]

File: src/schema/schema.graphql [hunk 2]
```diff
@@ -1310,6 +1310,7 @@ type AccountInfoAttrs {
 	zimbraFeatureMobileSyncEnabled: Boolean
 	zimbraFeatureRelatedContactsEnabled: Boolean
 	zimbraFeatureMailForwardingInFiltersEnabled: Boolean
+	zimbraPasswordAllowUsername: String
```
Comment (silentsakky) [inline src/schema/schema.graphql:1313]: should be boolean

Comment (sakshirawat0247) [reply src/schema/schema.graphql]: Yup, Changed it.Also figured out that we have a function coerceStringToBoolean which wasn't being used for resetPassword Response.So, added the same.

**Review by lavegupta — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

**Pull request merged**